src/styles/header.scss

```diff
@@ -0,0 +1,143 @@
+// @import "variables";
+
+// .header {
+//   font-family: "Roboto", sans-serif;
+//   display: flex;
+//   justify-content: space-between;
+//   align-items: center;
+//   background-color: $bg-color-header;
+//   margin: 0 auto;
+
+//   &__icon {
+//     display: flex;
+//     align-items: center;
+//     margin-left: $margin-header;
+//   }
+// }
+
+// .nav {
+//   margin-right: $margin-header;
+
+//   &__list {
+//     display: flex;
+//     height: 100%;
+//     margin: 0;
+//     padding: 0;
+//   }
+
+//   &__item {
+//     list-style: none;
+//     margin-right: 20px;
+//     position: relative;
+//   }
+
+//   &:last-child {
+//     margin: 0;
+//   }
+
+//   &__link {
+//     display: flex;
+//     align-items: center;
+//     text-decoration: none;
+//     color: #060B35;
+//     font-size: 12px;
+//     height: $height-header;
+//     text-transform: uppercase;
+
+//     &:hover {
+//       color: $header-color-active;
+//     }
+//   }
+// }
+
+// .header.is-active {
+//   color: $header-color-active;
+
+//   &::after {
+//     content: "";
+//     position: absolute;
+//     bottom: 0;
+//     left: 0;
+//     background-color: $header-color-active;
+//     height: 4px;
+//     border-radius: 8px;
+//     width: 100%;
+//   }
+// }
+
+// $header-color-active: #00acdc;
+// $height-header: 60px;
+// $margin-header: 50px;
+// $bg-color-header: #fff;
+@import "variables";
+
+body {
```

Comment:
You already set the styles for the `body` in another file, why are you duplicating it here?